src/languages/en.ts

```diff
@@ -610,6 +610,14 @@ export default {
         },
         waitingOnEnabledWallet: ({submitterDisplayName}: WaitingOnBankAccountParams) => `Started settling up, payment is held until ${submitterDisplayName} enables their Wallet`,
         enableWallet: 'Enable Wallet',
+        hold: 'Hold',
+        holdEducationalTitle: 'This request is on',
+        whatIsHoldTitle: 'What is hold?',
+        whatIsHoldExplain: 'Hold is our way of streamlining financial collaboration. "Reject" is so harsh!',
+        holdIsTemporaryTitle: 'Hold is usually temporary',
+        holdIsTemporaryExplain: "Because hold is used to clear up confusion or clarify an important detail before payment, it's not permanent.",
+        deleteHoldTitle: "Delete whatever won't be paid",
+        deleteHoldExplain: "In the rare case where something is put on hold and won't be paid, it's on the person requesting payment to delete it.",
```

Comment:
If you're adding these, you also need to add translations to the Spanish one.

Reply:
This is also why typecheck is failing

Reply:
And the unit tests as well 😁

Reply:
Of course, I was holding back on it and waiting for Robert to provide them.